gridstatus/isone_api/isone_api.py

```diff
@@ -210,7 +210,8 @@ def get_fuel_mix(
         mix_df.columns.name = None
 
         mix_df = mix_df.rename(columns={"BeginDate": "Time"})
-        mix_df["Time"] = pd.to_datetime(mix_df["Time"]).dt.strftime(
+        mix_df["Time"] = mix_df["Time"].apply(self.parse_problematic_datetime)
+        mix_df["Time"] = mix_df["Time"].dt.strftime(
```

Comment:
not sure i understand the call to `strftime`. Don't we normally return the pandas timestamp object here?

Reply:
sorry, i see you are still working. i thought i got a notification you rerequested review.

Reply:
yeah was trying some things with gs-etl which was running into problems with the datetimeindex of the interpolation. Working now.